h2o-py/h2o/utils/metaclass.py

```diff
@@ -186,6 +186,43 @@ def wrapper(*args, **kwargs):
 deprecated_fn = _DeprecatedFunction
 
 
+def deprecated_params_order(old_sig, is_called_with_old_sig=None):
```

Comment:
we can make param `is_called_with_old_sig` mandatory I guess as the decorator is useless otherwise